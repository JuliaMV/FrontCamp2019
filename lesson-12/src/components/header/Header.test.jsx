import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import Header from './Header';

Enzyme.configure({ adapter: new Adapter() });

describe('Header component', () => {
  test('should render with default props', () => {
    const component = shallow(<Header />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
