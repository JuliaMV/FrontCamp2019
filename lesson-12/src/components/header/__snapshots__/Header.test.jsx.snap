// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render with default props 1`] = `
<header>
  <div>
    <h1>
      <Logo />
    </h1>
    <h2>
      Find your movie
    </h2>
    <form
      action=""
      method="get"
      onSubmit={[Function]}
    >
      <input
        id="search-input"
        name="search-input"
        onChange={[Function]}
        type="text"
      />
      <button
        type="submit"
      >
        Search
      </button>
    </form>
    <div>
      <Filter
        activeFilter="title"
        fields={
          Array [
            "title",
            "genre",
          ]
        }
        filterHandler={[Function]}
        title="search by"
      />
    </div>
  </div>
</header>
`;
